### ISSUE TITLE
[Question] 

### ISSUE THREAD

**Issue by wjfwjfwjf** [haotian-liu/LLaVA#524]
### Question

hello，I want to know after pretrain, how much the loss is? what about after instruct tune?

**Comment by wjfwjfwjf** [comment]
thanks

**Comment by haotian-liu** [comment]
The logs are provided in MODEL ZOO: https://github.com/haotian-liu/LLaVA/blob/main/docs/MODEL_ZOO.md

Training logs: [wandb](https://api.wandb.ai/links/lht/6orh56wc).

Thanks.
